slides/scenarios/signup-page.ftd

```diff
@@ -0,0 +1,6 @@
+-- import: ui.fifthtry.com/slides/pages/signup
```

Comment:
You have created the new file, but have not added link to it in the top level `index.ftd`.